app/models/post.js

```diff
@@ -160,8 +160,8 @@ export default Model.extend(Comparable, ValidationEngine, {
     hasEmail: computed('email', 'emailOnly', function () {
         return this.email !== null || this.emailOnly;
     }),
-    willEmail: computed('emailRecipientFilter', function () {
-        return this.emailRecipientFilter !== null;
+    willEmail: computed('isScheduled', 'newsletter', function () {
+        return this.isScheduled && !!this.newsletter;
```

Comment:
The logic here wasn't quite correct, updated to reflect current main

```suggestion
    willEmail: computed('isScheduled', 'newsletter', 'email', function () {
        return this.isScheduled && !!this.newsletter && !!this.email;
```

Reply:
> Do you think we also should update the same check in HBS at app/components/editor-labs/modals/update-flow.hbs

Yeah, replacing `(and post.isScheduled post.newsletter (not post.email))` with `post.willEmail` is nicer 👌